app/static/src/scss/includes/_record.scss

```diff
@@ -209,6 +208,57 @@
   margin-top: -2rem;
 }
 
+.record-container .record-arrangement-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+
+  --ball-size: 13px;
+  --border-width: 3px;
+  --colour: #005ea5;
+}
+
+.record-container .record-arrangement-list li {
+  position: relative;
+  padding: 0 0 20px calc(var(--ball-size) + 1rem);
+  font-family: Arial, sans-serif;
+  font-size: 1rem;
+  line-height: 1.2rem;
+  text-wrap: wrap;
+}
+
+.record-container .record-arrangement-list li::before {
+  content: " ";
+  display: block;
+  position: absolute;
+  width: var(--ball-size);
+  height: var(--ball-size);
+  left: 0;
+  top: 0;
+  border-radius: 100%;
+  border: var(--border-width) solid var(--colour);
+  z-index: 2;
+}
+
+.record-container .record-arrangement-list li::after {
+  content: " ";
+  display: block;
+  position: absolute;
+  width: var(--border-width);
+  left: calc((var(--ball-size) / 2) + (var(--border-width) / 2));
+  top: calc(var(--ball-size) + var(--border-width));
+  bottom: -3px; /*  Minus the --border-width (can't make this negative in CSS so have had to hard-code it)  */
```

Comment:
apparently `calc(var(--border-width) * -1);` should work

Reply:
Added this change, thanks